catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/query/GeoNamesQueryLuceneDirectoryIndex.java

```diff
@@ -99,6 +99,12 @@ protected IndexSearcher createIndexSearcher(final IndexReader indexReader) {
     return doQuery(queryString, maxResults, directory);
   }
 
+  @Override
+  public List<String> getSuggestedNames(String queryString, int maxResults)
+      throws GeoEntryQueryException {
+    return null;
```

Comment:
Any reason why this returns `null` while the `GeoNamesWebService` implementation returns an empty list?

Reply:
This is intended to be a no-op for this impl of GeoIndexQueryable.

Reply:
I suggest we comment as such